### DISCUSSION TITLE
HTML Injection doesn't work on all pages

### DISCUSSION THREAD

**Discussion by IdkWhosThis** [requarks/wiki#3764] [Error / Bug Report]
I have written a JavaScript interval that changes the size of the site logo and removes the site title from the menu. For that, I have used a "Head HTML Injection". It works perfectly if I access a custom page that I have made in the Admin panel in Pages, but not on default pages like /t (tags browsing) or /p/profile (user profile), it simply doesn't inject. Even if I try to search for it in elements (F12, Elements, and then CTRL+F) on for example profile page, I get 0 results, but on the custom page, it finds it.

My code:
```html
<script>
function sleep(ms) {
        return new Promise(resolve => {
            setTimeout(resolve, ms)
        })
    }
changeInt = setInterval(async () => {
if(document.getElementsByClassName('v-avatar rounded-0')[0]) {
    document.getElementsByClassName('v-avatar rounded-0')[0].style.width = "90px";
}
if(document.getElementsByClassName("v-toolbar__title mx-3")[0]) {
    document.getElementsByClassName("v-toolbar__title mx-3")[0].remove()
}
console.clear();
}, 25);
async function clearInt() {
await sleep(5000);
await clearInterval(changeInt);
}
async function clearConsole() {
await sleep(1000);
await console.clear();
}
clearConsole();
clearInt();
</script>
```

**To Reproduce**

  1. Go to Admin panel > Theme
  2. Paste my script into "Head HTML Injection"
  3. Save it
  4. Go to any custom page, where it works and see the result
  5. Go to the user's profile page or tag browsing and see that it doesn't inject into these pages
 
**Expected behavior**
The "Head HTML Injection" works on all custom and all default/system pages.

**Host Info (please complete the following information):**

- OS: Linux - Ubuntu (bionic) 20.04 x64
- Wiki.js version: latest (2.5.191)
- Database engine: MySQL 14.14
- Node: v12.11.1

**Comment by sandzhaj** [comment]
Same here. But I used
```
<style>
.v-avatar.rounded-0 { height: 44px !important; min-width: 34px; width: 72px !important;}
</style>
```
It also works perfectly if I access a custom page. Any solutions to make it working on the default pages?
